ccswbs/doorknob
# Create page to replace uoguelph.ca/ens-test

This PR adds a duplicate of the home page meant for testing App Armor Alerts and will replace the existing page at [https://uoguelph.ca/ens-test](https://uoguelph.ca/ens-test)

# Summary of changes

## Frontend
- Added `pages/ens-test.js`
- Updated `appArmorAlert.js` component to use parameters for alert id rather than using env variables

### Incidental changes
- Removed a bit of code from `gatsby-config.js` which set the previously mentioned env variable
- Replaced React Helmet with Gatsby Script in the App Armor and BrightEdge components

## Backend
- N/A

## Test plan
- PR build: https://doorknobmain-enstestpage.gatsbyjs.io/

1. Request @RazaMM or someone from the C&M web team to post a test alert.
1. Go to https://doorknobmain-enstestpage.gatsbyjs.io/ens-test
1. Verify the alert appears at the top of the page when loaded (Note it may take a few seconds for it to appear)
